core/SymbolRef.h

```diff
@@ -235,143 +235,149 @@ class Symbols {
         return SymbolRef(nullptr, 41);
     }
 
-    // Used to mark the presence of an ancestor that we were unable to
-    // statically resolve to a class or module
-    static SymbolRef StubAncestor() {
+    // Used to mark the presence of an mixin that we were unable to
+    // statically resolve to a module
+    static SymbolRef StubMixin() {
         return SymbolRef(nullptr, 42);
     }
 
-    static SymbolRef T_Enumerable() {
+    // Used to mark the presence of a superclass that we were unable to
+    // statically resolve to a class
+    static SymbolRef StubSuperClass() {
```

Comment:
wow you're brave. I always just add to the end. Some day we should somehow fix this...

Reply:
if I get another merge conflict I will do that :p